File: atividade-7/atividade.py
class Professor:
    def __init__(self, nome):
        self.nome = nome

class Escola:
    def __init__(self, nome):
        self.nome = nome
        self.professores = []

    def adicionar_professor(self, professor):
        self.professores.append(professor)

    def exibir_professores(self):
        print(f"Professores da escola {self.nome}:")
        for professor in self.professores:
            print(f"- {professor.nome}")

escola = Escola("Escola A")
professor1 = Professor("João")
professor2 = Professor("Maria")

escola.adicionar_professor(professor1)
escola.adicionar_professor(professor2)
escola.exibir_professores()

File: atividade-7/atividade.go
package main

import "fmt"

type Professor struct {
	Nome string
}

type Escola struct {
	Nome        string
	Professores []Professor
}

func (e *Escola) AdicionarProfessor(professor Professor) {
	e.Professores = append(e.Professores, professor)
}

func (e Escola) ExibirProfessores() {
	fmt.Printf("Professores da escola %s:\n", e.Nome)
	for _, professor := range e.Professores {
		fmt.Printf("- %s\n", professor.Nome)
	}
}

func main() {
	escola := Escola{Nome: "Escola A"}
	professor1 := Professor{Nome: "João"}
	professor2 := Professor{Nome: "Maria"}

	escola.AdicionarProfessor(professor1)
	escola.AdicionarProfessor(professor2)
	escola.ExibirProfessores()
}
